lib/auth/init.go

```diff
@@ -274,6 +275,10 @@ func Init(cfg InitConfig, opts ...ServerOption) (*Server, error) {
 		return nil, trace.Wrap(err)
 	}
 
+	if err := initSetDefaultInstaller(ctx, asrv); err != nil {
+		return nil, trace.Wrap(err)
+	}
```

Comment:
Why do we need to set anything in the backend? Why not just do this:

- When installer is being retrieved, try to fetch it from the backend.
 - If found, return it.
 - If not found, return a default installer resource (just build the resource in memory).

Then we won't need to save any singletons in the backend and won't need to worry about re-saving the default installer resource when user deletes their own. Does that make sense?